File: ClinicControlCenter/Domain/DTOs/UserDTO.cs
namespace ClinicControlCenter.Domain.DTOs
{
    public class UserDTO
    {
        #region IdentityBase

        public string UserName { get; set; }

        public string Email { get; set; }

        #endregion

        public string FullName { get; set; }

        public string Telephone { get; set; }

        public string CEP { get; set; }

        public string Street { get; set; }

        public string Neighborhood { get; set; }

        public string City { get; set; }

        public string State { get; set; }

        #region Optional
        public string Password { get; set; }

        #endregion

    }
}

File: ClinicControlCenter/ClientApp/src/app/system-management/models/userDTO.ts
export class UserDTO {
  Password?: string;
  UserName?: string;
  Email: string;
  FullName: string;
  Telephone: string = "";
  CEP: string = "";
  Street: string = "";
  Neighborhood: string = "";
  City: string = "";
  State: string = "";

  constructor(obj?: Partial<UserDTO>) {
    if (obj)
      for (const prop of Object.keys(obj)) {
        this[prop] = obj[prop];
      }
  }
}
